.github/workflows/ci.yml

```diff
@@ -72,7 +72,9 @@ jobs:
       - name: Run unittest tests with coverage
         run: |
           pytest --cov=autogpt --without-integration --without-slow-integration --cov-report term-missing --cov-branch --cov-report xml --cov-report term
-
+          pytest --cov=autogpt  tests/integration/goal_oriented --cov-report term-missing --cov-branch --cov-report xml --cov-report term
```

Comment:
The other includes this one I believe. May want to verify

Reply:
Wait it doesn’t because without integration. May be worth it to just remove the without integration flag

Reply:
we can keep it, this way we first run unit tests then pay the money for integration tests

Reply:
If we setup the test order correctly, we should be able to do that with the one command

Reply:
Looking into it now

Reply:
There’s ways but we can probably just run it and worry about that later